intake_parquet/source.py

```diff
@@ -106,5 +106,32 @@ def _to_dask(self):
         self._load_metadata()
         return self._df
 
+    def to_spatial(self):
+        """
+        Create a datashader spatial object from the parquet data
+        """
+        try:
+            from datashader.spatial.points import SpatialPointsFrame
+        except ImportError:
+            raise ImportError('SpatialPointsFrame not found in this '
+                'version of datashader. Get latest using '
+                '`conda install -c pyviz datashader`.')
+        import json
+        import fastparquet as fp
+
+        frame = self._df or self.to_dask()
+        urlpath =  self._get_cache(self._urlpath)[0]
+        pf = fp.ParquetFile(urlpath)
+        # Check for spatial points metadata
+        if 'SpatialPointsFrame' in pf.key_value_metadata:
+            # Load metadata
+            props = json.loads(pf.key_value_metadata['SpatialPointsFrame'])
+        else:
+            props = None
+
+        # Call DataFrame constructor with the internals of frame
+        return SpatialPointsFrame(frame.dask, frame._name, frame._meta,
```

Comment:
Presumably this can fail in various way?
I suppose the point is to pass this immediately to graphics, would it make sense to (optionally) construct a datashader/holoviews object right here?

Reply:
Maybe I should have made this clearer, I copied all of this straight from: https://github.com/pyviz/datashader/blob/master/datashader/spatial/points.py#L297-L312

Reply:
OK, so we just defer everything to whatever that interface does... I guess then the docstring should be caveated to say that whatever goes wrong, it's not Intake's fault!